Lib/test/test_xmlrpc.py

```diff
@@ -1307,10 +1398,11 @@ def test_xmlrpcserver_has_use_builtin_types_flag(self):
 
 @support.reap_threads
 def test_main():
-    support.run_unittest(XMLRPCTestCase, HelperTestCase, DateTimeTestCase,
-            BinaryTestCase, FaultTestCase, UseBuiltinTypesTestCase,
-            SimpleServerTestCase, SimpleServerEncodingTestCase,
-            KeepaliveServerTestCase1, KeepaliveServerTestCase2,
+    support.run_unittest(XMLRPCTestCase, SimpleXMLRPCDispatcherTestCase,
```

Comment:
I think it would be better to add new test at the end of the list of tests or at separate line. This would make the diff smaller and easier fore review.